VulkanHppGenerator.cpp

```diff
@@ -23,14 +23,6 @@
 #include <regex>
 #include <sstream>
 
-struct MacroData
```

Comment:
I've moved this to the `.hpp` file, because this was hanging around here weirdly, and it made more sense to stay with the rest of the structs in the header.